### DISCUSSION TITLE
AzureMonitor: Add filters to resource picker

### DISCUSSION THREAD

**Discussion by aangelisc** [grafana/grafana#60681] [Data Sources]
The Azure Portal supports filtering by subscription, resource type, and location. This should also be supported in the datasource resource picker as it makes it far easier to find resources if you don't remember the name or exact path to the resource.
![image](https://user-images.githubusercontent.com/15019026/209132038-04553ce7-c40d-433d-8b96-15a9526ae4b5.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
